_config.yml

```diff
@@ -49,3 +49,5 @@ defaults:
       category: posts
 
 # For local work: bundle exec jekyll serve --watch --baseurl '/proghistdev' or wherever else your local site is located
+
+theme: jekyll-theme-slate
```

Comment:
So, one thing with the jekyll theme -- it appears that you are adding an alternative theme to the config file. The lesson will need to work with the default theming.